src/Common/src/Interop/Unix/libssl/Interop.libssl.msbuild

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemGroup>
```

Comment:
@kapilash is removing the .msbuild though I still don't see a downside to this. I believe all these files are required.